src/templates/docs-page.tsx

```diff
@@ -41,6 +41,11 @@ const DocsPageTemplate = ({
         isWidthLimited
         padding={{ default: "padding" }}
       >
+        <Alert variant="info" title="Important Notice" isInline className="pf-u-mb-xl">
+          <p>
+            Please note that the latest information about release notes has been moved <a href="https://github.com/opendatahub-io/opendatahub-operator/releases/latest" target="_blank" rel="noopener noreferrer">here</a>.
```

Comment:
maybe something more like if want to checkout previous v2 releases, go https://github.com/opendatahub-io/opendatahub-operator/releases and use "find release" to search for?

Reply:
yep I could do that

Reply:
done